utils/proxy/_deserializer.py

```diff
@@ -96,7 +93,8 @@ def deserialize_dd_appsec_s_meta(key, payload):
     try:
         return json.loads(gzip.decompress(base64.b64decode(payload)).decode())
     except Exception as e:
-        raise ValueError(f"meta {key} should be b64/gziped JSON.\nPayload: {payload}") from e
+        # b64/gzip is optional
+        return payload
```

Comment:
is payload a json-string ? if yes, can you decode it ?

Reply:
nice catch, I need to fix that.

Reply:
fixed